frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -204,7 +242,11 @@ export const personsLogic = kea<personsLogicType<PersonPaginatedResponse>>({
         },
         '/person/*': ({ _: person }, { sessionRecordingId }, { activeTab }) => {
             if (sessionRecordingId) {
-                actions.navigateToTab(PersonsTabType.SESSIONS)
+                if (values.showSessionRecordings) {
+                    actions.navigateToTab(PersonsTabType.SESSION_RECORDINGS)
+                } else if (values.showSessions) {
```

Comment:
I think you can replace this else if check with an else so that we always default to navigating to sessions in any other scenario. For example, if for some reason `!!featureFlags[FEATURE_FLAGS.REMOVE_SESSIONS] && !currentTeam?.session_recording_opt_in === true`, this is a noop